modules/bmo-performing-a-live-update-to-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -45,16 +60,16 @@ The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `serv
 $ oc annotate bmh <hostname> reboot.metal3.io=""
```

Comment:
[source,terminal]
----
$ oc drain <hostname> --force
----
+
Power off the node for a period of approximately five minutes. This will ensure any infrastructure pods that may still running on the node are marked as offline by their respective daemonsets and/or controllers and any requests will be handled by the other nodes:

oc patch bmh/<hostname> --type merge -p  '{"spec": {"online": false}}'

( wait 5 minutes )

oc patch bmh/<hostname> --type merge -p  '{"spec": {"online": true}}'

Once the BareMetalHost starts booting back up, servicing operation commences and the Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.